services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/ApplicationConfig.java

```diff
@@ -261,6 +262,17 @@ public MongoDbBasedRegistrationConfigProperties serviceProperties() {
         return new MongoDbBasedRegistrationConfigProperties();
     }
 
+    /**
+     * Gets properties for the hono tenant mongodb service.
+     *
+     * @return The properties.
+     */
+    @Bean
+    @ConfigurationProperties(prefix = "hono.tenant.mongodb")
+    public MongoDbBasedTenantsConfigProperties mongoDbTenantsProperties() {
```

Comment:
IMHO we could use the same naming convention as file based device registry which is `tenantsProperties()`

Reply:
resolved